release-to-candidate/action.yaml

```diff
@@ -65,6 +69,8 @@ runs:
       id: build
       shell: bash
       env:
+        SNAPCRAFT_REMOTE_BUILD_STRATEGY: "disable-fallback"
```

Comment:
Interesting! Never seen this - are there any docs or forum posts about this?

Reply:
Okay found it - it seems this is the default for `core20` and `core22` anyway - but leaving it here for our `core18` stuff seems reasonable

Reply:
> but leaving it here for our `core18` stuff seems reasonable

I am sorry, didn't understand this, should I drop this? In that case we'll not be able to use new keywords. Also, I have some PRs ready, which will port many widely used apps to core22.

Reply:
My point is: anything using `core20` or `core22` is already using this behaviour by default - so this change will only affect `core18` snaps - which is fine :)

Reply:
Actually not. See here, it doesn't use that

https://github.com/soumyaDghosh/ffmpeg-sdk/actions/runs/8306691603/job/22735074775#step:2:179

This is `snapcraft 8.0.4` and it still didn't allow me to use the `lint` keyword.